presto-pinot-toolkit/pom.xml

```diff
@@ -383,7 +383,7 @@
         <dependency>
             <groupId>io.perfmark</groupId>
             <artifactId>perfmark-api</artifactId>
-            <version>0.23.0</version>
+            <version>0.27.0</version>
```

Comment:
This change doesn't look related to the PR & commit description. And current version of this dependency doesn't seem to have CVEs https://mvnrepository.com/artifact/io.perfmark/perfmark-api/0.23.0 
So curious, why are we upgrading this?

Reply:
This is not required. I will revert it. My changes have been lost due to resetting the Head 1 commit ago.

> This change doesn't look related to the PR & commit description. And current version of this dependency doesn't seem to have CVEs https://mvnrepository.com/artifact/io.perfmark/perfmark-api/0.23.0 So curious, why are we upgrading this?

Reply:
Uploading screenshot for Reference.
<img width="918" alt="Screenshot 2024-11-15 at 5 44 29 PM" src="https://github.com/user-attachments/assets/91a9d416-c2cd-486a-bd8d-85d7a5e1a287">